lib/sinatra_sms_takeaway.rb

```diff
@@ -0,0 +1,27 @@
+require 'sinatra'
```

Comment:
Nice attempt, you might want to try re-writing this implementation after completing week 3.